### ISSUE TITLE
Incompatibility with the results

### ISSUE THREAD

**Issue by chikubee** [hanxiao/bert-as-service#440]
**Prerequisites**

> Please fill in by replacing `[ ]` with `[x]`.

* [x] Are you running the latest `bert-as-service`?
* [x] Did you follow [the installation](https://github.com/hanxiao/bert-as-service#install) and [the usage](https://github.com/hanxiao/bert-as-service#usage) instructions in `README.md`?
* [x] Did you check the [FAQ list in `README.md`](https://github.com/hanxiao/bert-as-service#speech_balloon-faq)?
* [x] Did you perform [a cursory search on existing issues](https://github.com/hanxiao/bert-as-service/issues)?

I've fine-tuned bert base uncased model for Text Classification, I followed the issues relevant to fetching the final predicted probabilities by changing the server-graph.

If I am fetching the output layer just how i had done during training using model.get_pooled_output() and thereby computing the logits using output_weights and output_bias.,
I've used the same ckpt file during serving but results are different.

Any leads so as to why this could be happening will be helpful.
Thanks in advance
